### PR TITLE
Assignment 4 developement

### PR DESCRIPTION
**Pull request by nitishnivedan**

### CODE REVIEW

**Comment by iswarya15** [comment]
Reviewed. All looks good.

**Review by iswarya15 — COMMENTED** [review]
Reviewed. All looks good.

**Comment by shaksham08** [comment]
The code looks good

**Review by shaksham08 — APPROVED** [review]

File: src/Assignment4/CardComponent/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import styles from "./styles-card.css";
```
Comment (ap221882) [inline src/Assignment4/CardComponent/index.js]: Do you need styles here? Shouldn't it be only <import "./styles-card.css"> Or you're trying a CSS modular import?

File: src/Assignment4/CardComponent/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import logo from "../../logo/usericon.png";
```
Comment (ap221882) [inline src/Assignment4/CardComponent/index.js]: logo is unused in this component

File: src/Assignment4/Header/index.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import "./styles.css";
+import logo from "../../logo/React.webp";
+import usericon from "../../logo/usericon.png";
+import searchicon from "../../logo/searchicon.png";
+
+const Header = ({}) => (
```
Comment (ap221882) [inline src/Assignment4/Header/index.js:6]: You can remove the empty curly braces

**Review by ap221882 — COMMENTED** [review]
Reviewed, nicely structured.

File: src/Assignment4/App.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import ReactDOM from "react-dom/client";
+
+import RootElement from "./Root";
+
+const root = ReactDOM.createRoot(document.getElementById("root"));
+root.render(
+  <>
```
Comment (smriti-porwal) [inline src/Assignment4/App.js]: The Fragment <></> is not needed here so you can remove it.

File: src/Assignment4/CardComponent/index.js (hunk 1 above)
Comment (nitishnivedan) [reply src/Assignment4/CardComponent/index.js]: yes you are correct here only import "./styles-card.css is needed. I have removed the styles part

File: src/Assignment4/Header/index.js (hunk 3 above)
Comment (nitishnivedan) [reply src/Assignment4/Header/index.js:6]: this empty curly braces is fine. its like the developer is ready for any future de structuring. doesnt have any impact.

File: src/Assignment4/App.js (hunk 4 above)
Comment (nitishnivedan) [reply src/Assignment4/App.js]: removed.

**Pull request merged**